iconservice/icon_service_engine.py

```diff
@@ -494,10 +494,12 @@ def _process_issue_transaction(self,
         treasury_address: 'Address' = self._icx_engine.fee_treasury_address
         tx_result = TransactionResult(context.tx, context.block)
         tx_result.to = treasury_address
-
+        # temp data
+        i_score = 1_000_000
```

Comment:
Is it correct value for every case?
If you plan to modify later, add TODO comments

Reply:
That is just temporary data. I have commented todo about this.